crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -1078,7 +1079,7 @@ fn get_task_time(state: &JobTaskState) -> (Option<DateTime<Utc>>, Option<DateTim
             end_date,
             ..
         } => (Some(started_data.start_date), Some(*end_date)),
-        JobTaskState::Canceled { started_data: None } | JobTaskState::Waiting => (None, None),
+        _ => (None, None),
```

Comment:
Let's remove the `_` case, so that if new states are added the compiler will tell us to update this function.